postinstall.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var process = require('process');
+var path = require('path');
+var VERSION = require('./node_modules/electron/package.json').version;
+var gypPath = path.join(__dirname, 'node_modules/.bin') + "/node-gyp";
```

Comment:
ironically this will not work on windows because you have to path.join the `/` as well